components/x-live-blog-post/src/Timestamp.jsx

```diff
@@ -27,19 +27,19 @@ export default ({ publishedTimestamp }) => {
 			<time
 				data-o-component="o-date"
 				className={`o-date ${styles['live-blog-post__timestamp']}`}
-				itemProp="datePublished"
+				dateTime={publishedTimestamp}
 				data-o-date-format={format}
-				dateTime={publishedTimestamp}>{formatted}
+				itemProp="datePublished">
+					<span data-o-date-printer>{formatted}</span>
```

Comment:
FYI, this `{formatted}` date is overridden by `o-date` and only appears if JS is disabled:

![image](https://user-images.githubusercontent.com/30316203/89403857-e898d480-d710-11ea-9ded-1c10236d3d12.png)

Reply:
Yep, I think this looks good to me